openedx/frontend-lib-content-components
# fix: override selectablebox with copy of version from edx paragon 21.5.6

## Description

We are encountering a bug in our stage environment that is very hard to reproduce locally, but not impossible: When you create a new problem and try to select a different problem type, this fails.

This is the same bug dealt with in several previous PRs.
Since I was able to reproduce it locally, it is definitely not just a bug affecting only 2U-specific things.

### Expected behavior
- create a new problem block
- you should be able to select a different problem type
- the created block should have the new selected problem type

### Actual behavior on stage environment
- you cannot select any problem type
- all problems default to Single Select

### Previous steps

Previously, I reproduced this locally by just adding the latest version of SelectableBox as a copy into this repo and importing it from there. Then, under the mistaken assumption that there was a missing context provider, I added that and it got fixed locally. However it turned out to not work on stage.

### Measures taken in this PR

- I replaced the entire `SelectableBox` component with all subcomponents with a version from @edx/paragon@21.5.6, which was apparently the version in the commit that didn't have the error yet. This fixes the problem on stage from what I observed when doing the same thing in course-authoring.

## Testing instructions

- You'll likely not be able to reproduce this bug locally. But just make sure that the problem type select still works correctly (that I haven't broken any existing functionality by accident).
